vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -23908,3 +23908,70 @@ vaem:GMD_QUDT-QUANTITY-KINDS-ALL
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "QUDT Quantity Kind Vocabulary Metadata Version 2.1.33" ;
 .
+
+
+
+quantitykind:ElectricityCost a qudt:QuantityKind;
```

Comment:
My impression is that this quantity kind is not specific to electricity, and it does not clarify the "unit of electricity" anyway (power versus energy). A better name and description would be CostPerUnitEnergy (as distinct from Power). Thoughts?